pyanaconda/subscription.py

```diff
@@ -0,0 +1,32 @@
+#
+# Subscription related helper functions.
+#
+# Copyright (C) 2020  Red Hat, Inc.  All rights reserved.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+from pyanaconda.modules.common.constants.namespaces import SUBSCRIPTION_NAMESPACE
+from pyanaconda.modules.common.constants.services import BOSS
+
+
+def check_subscription_module_available():
```

Comment:
Maybe I would create a more general function `is_module_available` and put it to `pyanaconda.modules.common.utils`. The function could have one argument of the type `DBusServiceIdentifier`. Then we could use it like this:

```
is_module_available(SUBSCRIPTION)
```

Reply:
Sounds good, I'll do that.

Reply:
Using `util` for consistency in the end as we already have `core/util.py`.